Engine/ResourceManagement/Importer/Importer.cpp

```diff
@@ -54,10 +56,20 @@ bool Importer::ImportRequired(const Path& file_path)
 
 		Metafile* metafile = App->resources->metafile_manager->GetMetafile(*metafile_path);
 		assert(App->resources->metafile_manager->IsMetafileConsistent(*metafile));
-		return
-			metafile_path->GetModificationTimestamp() < file_path.GetModificationTimestamp()
-			|| metafile->version < Importer::IMPORTER_VERSION
-			|| !App->filesystem->Exists(metafile->exported_file_path);
+
+		if (metafile->version < Importer::IMPORTER_VERSION || !App->filesystem->Exists(metafile->exported_file_path))
+		{
+			return true;
+		}
+		else if (App->filesystem->Exists(metafile->exported_file_path))
+		{
+			Path* library_path = App->filesystem->GetPath(metafile->exported_file_path);
```

Comment:
I think this pointer is not being deleted

Reply:
It is, all `Path` pointers are deleted in the destruction of + ModuleFileSystem`